kotlin/toggle-between-feature-request-modes/src/main/java/com/esri/arcgisruntime/sample/togglebetweenfeaturerequestmodes/MainActivity.kt

```diff
@@ -0,0 +1,242 @@
+/* Copyright 2022 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.togglebetweenfeaturerequestmodes
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.ProgressBar
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
+import com.esri.arcgisruntime.data.QueryParameters
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import com.esri.arcgisruntime.data.ServiceFeatureTable.FeatureRequestMode
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.BasemapStyle
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.DrawStatus
+import com.esri.arcgisruntime.mapping.view.DrawStatusChangedEvent
+import com.esri.arcgisruntime.mapping.view.MapView
+import com.esri.arcgisruntime.sample.togglebetweenfeaturerequestmodes.databinding.ActivityMainBinding
+import java.util.Collections
+import java.util.concurrent.atomic.AtomicInteger
+
+
+class MainActivity : AppCompatActivity() {
+
+    private val TAG: String = MainActivity::class.java.simpleName
+
+    private val activityMainBinding by lazy {
+        ActivityMainBinding.inflate(layoutInflater)
+    }
+
+    private val mapView: MapView by lazy {
+        activityMainBinding.mapView
+    }
+
+    private val modeTV: TextView by lazy {
+        activityMainBinding.modeTV
+    }
+
+    private val populateTV: TextView by lazy {
+        activityMainBinding.populateTV
+    }
```

Comment:
can we make these explicitly buttons rather than textviews? Took me a minute to find them in code because I was looking for buttons--might save our users some friction there